pkg/apis/trainer/v1alpha1/trainingruntime_types.go

```diff
@@ -210,13 +216,15 @@ type MPIMLPolicySource struct {
 
 	// Implementation name for the MPI to create the appropriate hostfile.
 	// Defaults to OpenMPI.
+	// +kubebuilder:default="OpenMPI"
 	MPIImplementation *MPIImplementation `json:"mpiImplementation,omitempty"`
 
 	// Directory where SSH keys are mounted.
 	SSHAuthMountPath *string `json:"sshAuthMountPath,omitempty"`
```

Comment:
@akshaychitneni Let's make this as defaults to `/root/.ssh`, similar as MPI Operator.